wdl-lint/tests/lints/expression-spacing/source.errors

```diff
@@ -1,167 +1,183 @@
 
 note[ExpressionSpacing]: operators must be followed by whitespace
-   ┌─ tests/lints/expression-spacing/source.wdl:40:29
+   ┌─ tests/lints/expression-spacing/source.wdl:43:29
    │
-40 │                             ||!e
+43 │                             ||!e
    │                             ^^
    │
    = fix: add a space after this operator
 
 note[ExpressionSpacing]: multi-line if...then...else must have a preceding parenthesis and newline
-   ┌─ tests/lints/expression-spacing/source.wdl:56:21
+   ┌─ tests/lints/expression-spacing/source.wdl:59:21
    │
-56 │         Boolean v = if 
+59 │         Boolean v = if 
    │                     ^^
    │
    = fix: add a open parenthesis and newline prior to this if
 
 note[ExpressionSpacing]: multi-line if...then...else must have a preceding space
-   ┌─ tests/lints/expression-spacing/source.wdl:57:15
+   ┌─ tests/lints/expression-spacing/source.wdl:60:15
    │
-57 │         a < b then true
+60 │         a < b then true
    │               ^^^^
    │
    = fix: add a newline before the then keyword
 
 note[ExpressionSpacing]: multi-line if...then...else must have a following newline and parenthesis
-   ┌─ tests/lints/expression-spacing/source.wdl:58:9
+   ┌─ tests/lints/expression-spacing/source.wdl:61:9
    │
-58 │         else false
+61 │         else false
    │         ^^^^
    │
    = fix: add a newline and close parenthesis after to this else clause
 
 note[ExpressionSpacing]: multi-line if...then...else must have a preceding space
-   ┌─ tests/lints/expression-spacing/source.wdl:63:22
+   ┌─ tests/lints/expression-spacing/source.wdl:66:22
    │
-63 │             if a < b then true
+66 │             if a < b then true
    │                      ^^^^
    │
    = fix: add a newline before the then keyword
 
 note[ExpressionSpacing]: multi-line if...then...else must have a preceding space
-   ┌─ tests/lints/expression-spacing/source.wdl:68:22
+   ┌─ tests/lints/expression-spacing/source.wdl:71:22
    │
-68 │             if a < b then true
+71 │             if a < b then true
    │                      ^^^^
    │
    = fix: add a newline before the then keyword
 
 note[ExpressionSpacing]: multi-line array/map/object literals must have a newline following the opening token
-   ┌─ tests/lints/expression-spacing/source.wdl:78:21
+   ┌─ tests/lints/expression-spacing/source.wdl:81:21
    │
-78 │         Boolean i = [1
+81 │         Boolean i = [1
    │                     ^
    │
    = fix: add a newline after this token
 
 note[ExpressionSpacing]: multi-line array/map/object literals must have a newline preceding the closing token
-   ┌─ tests/lints/expression-spacing/source.wdl:80:17
+   ┌─ tests/lints/expression-spacing/source.wdl:83:18
    │
-80 │             ,2,3] == [1,2,4]
-   │                 ^
+83 │             ,2,3,] == [1,2,4]
+   │                  ^
    │
    = fix: add a newline before this token
 
+note[CommentWhitespace]: comment not sufficiently indented
```

Comment:
I think we should leave these, as they are related to the diagnostic being tested. `CommentWhitespace` is firing because expression spacing is off. It's not ideal, and I think we can eventually use some code from `wdl-format` (or `wdl-edit` or whatever we're calling it) to make this Rule act a bit nicer.